README.md

```diff
@@ -1,5 +1,84 @@
-# beman.inplace\_vector: A dynamically-resizable vector with fixed capacity and embedded storage
-
 <!--
 SPDX-License-Identifier: <SPDX License Expression>
 -->
+
+# beman.inplace\_vector
+
+A dynamically-resizable vector with fixed capacity and embedded storage
+
+This repository implements `std::inplace_vector` extension targeting C++26.
+The `beman.inplace_vector` library aims to evaluate the stability, 
+the usability, and the performance of these proposed changes before they 
+are officially adopted by WG21 into the C++ Working Draft. 
+Additionally, it allows developers to use these new features before they 
+are implemented in major standard library compilers.
+
+## Implements
+
+- [`inplace_vector` P0843R14](https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2024/p0843r14.html)
```

Comment:
I am not sure if this is the revision this repo is updated to.

Reply:
That's the target anyway.

Reply:
Alright!